gammapy/irf/effective_area_table.py

```diff
@@ -542,6 +545,8 @@ def evaluate(self, offset=None, energy=None):
             val = self._eval_spline(offset.value, np.log10(energy.value))
         else:
             raise ValueError('Invalid interpolation method: {}'.format(method))
+
+        val = np.reshape(val, np.append(offset_shape, energy_shape))
```

Comment:
@tibaldo mentioned that the interpolation didn't work for his use case: a 2D or 3D array of offset values, needed to evaluate effective area on an image or cube to make exposure images.

I didn't try myself, but @joleroi, it could be that the interpolation function doesn't work for such inputs, no?